.github/workflows/config/spelling_allowlist.txt

```diff
@@ -263,4 +272,4 @@ upvote
 variadic
 variational
 verifier
-workflow
+workflow
```

Comment:
Could be nothing, but, confirm there aren't odd whitespace / line ending changes.

Reply:
Fixed in [a3964e2](https://github.com/NVIDIA/cuda-quantum/pull/1763/commits/a3964e284910f99cbbddddfa7493747ab9735855). I think it was probably a whitespace, but I couldn't find anything odd here.